service/src/main/kotlin/fi/espoo/evaka/emailclient/EvakaEmailMessageProvider.kt

```diff
@@ -55,78 +80,139 @@ class EvakaEmailMessageProvider : IEmailMessageProvider {
             You have an unanswered decision from Espoo’s early childhood education. The decision must be accepted or rejected within two weeks of receiving it.
             </p>
             <p>
-            The person who submitted the application can accept or reject an unanswered decision by logging in to <a href="espoonvarhaiskasvatus.fi">espoonvarhaiskasvatus.fi</a> or by sending the completed form on the last page of the decision to the address specified on the page.
+            The person who submitted the application can accept or reject an unanswered decision by logging in to <a href="https://espoonvarhaiskasvatus.fi">espoonvarhaiskasvatus.fi</a> or by sending the completed form on the last page of the decision to the address specified on the page.
             </p>
             <p>
             You cannot reply to this message. If you have questions, please contact early childhood education service counselling, tel. 09 816 31000.
             </p>
         """
-            .trimIndent()
-    }
-
-    override fun getPendingDecisionEmailText(): String {
-        return """
-            Sinulla on vastaamaton päätös Espoon varhaiskasvatukselta. Päätös tulee hyväksyä tai hylätä kahden viikon sisällä sen saapumisesta.
-
-            Hakemuksen tekijä voi hyväksyä tai hylätä vastaamattomat päätökset kirjautumalla osoitteeseen https://espoonvarhaiskasvatus.fi, tai palauttamalla täytetyn lomakkeen päätöksen viimeiseltä sivulta siinä mainittuun osoitteeseen.
-
-            Tähän viestiin ei voi vastata. Tarvittaessa ole yhteydessä varhaiskasvatuksen palveluohjaukseen p. 09 816 31000
-
-            -----
+                    .trimIndent()
+        )
 
-            Du har ett obesvarat beslut av småbarnspedagogiken i Esbo. Beslutet ska godkännas eller förkastas inom två veckor från att det inkommit.
+    private val applicationReceivedSubject =
+        "Olemme vastaanottaneet hakemuksenne / Vi har tagit emot din ansökan / We have received your application"
 
-            Den som lämnat in ansökan kan godkänna eller förkasta obesvarade beslut genom att logga in på adressen https://esbosmabarnspedagogik.fi eller genom att returnera den ifyllda blanketten som finns på sista sidan av beslutet till den adress som nämns på sidan.
+    override fun clubApplicationReceived(language: Language): EmailContent =
+        EmailContent(
+            subject = applicationReceivedSubject,
+            text =
+                """
+            Hyvä(t) huoltaja(t),
 
-            Detta meddelande kan inte besvaras. Kontakta vid behov servicehandledningen inom småbarnspedagogiken, tfn 09 816 27600
+            Lapsenne kerhohakemus on vastaanotettu.Hakemuksen tehnyt huoltaja voi muokata hakemusta osoitteessa www.espoonvarhaiskasvatus.fi siihen saakka, kunnes se on otettu käsittelyyn.
```

Comment:
Insert space:: vastaanotetttu.Hakemuksen -> vastaanotetttu. Hakemuksen. Also should it be https://espoonvarhaiskasvatus.fi instead of www.espoonvarhaiskasvatus.fi